chromadb/db/mixins/embeddings_queue.py

```diff
@@ -85,7 +91,22 @@ def __init__(self, system: System):
         self._subscriptions = defaultdict(set)
         self._max_batch_size = None
         self._opentelemetry_client = system.require(OpenTelemetryClient)
+        self._batch_ingestion = system.settings.require("batch_ingest")
         super().__init__(system)
+        if self._batch_ingestion:
+            t = threading.Thread(target=self._batch_worker, daemon=True)
```

Comment:
If this thread dies who brings it back

Reply:
That's a great point. I think we can look into #1691 for a possible solution to this.